### PR TITLE
feat: Pretty print JSON for python. Fix missing regular expressions for masking out ephemeral information

### PR DESCRIPTION
**Pull request by astuyve**
### What does this PR do?
- Prints JSON with indentations and newlines to increase readability
- Mask out ephemeral data which caused integration tests to always fail

### Types of Changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [ ] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog
- [ ] This PR passes the integration tests (ask a Datadog member to run the tests)

### CODE REVIEW

File: tests/integration/serverless.yml [hunk 1]
```diff
@@ -17,7 +17,6 @@ provider:
     DD_TRACE_ENABLED: true
     DD_API_KEY: ${env:DD_API_KEY}
     DD_TRACE_MANAGED_SERVICES: true
-    DD_CAPTURE_LAMBDA_PAYLOAD: true
```
Comment (nhinsch) [inline tests/integration/serverless.yml:20]: Why did you remove this?

File: scripts/run_integration_tests.sh [hunk 2]
```diff
@@ -175,14 +175,15 @@ for handler_name in "${LAMBDA_HANDLERS[@]}"; do
         # Replace invocation-specific data like timestamps and IDs with XXXX to normalize logs across executions
         logs=$(
             echo "$raw_logs" |
+                node parse-json.js |
```
Comment (nhinsch) [inline scripts/run_integration_tests.sh:178]: 🙏

**Comment by nhinsch** [comment]
Looks good, how were tests passing before if there were so many values that needed to be masked?

**Review by jcstorms1 — APPROVED** [review]
👍 LGTM

**Comment by astuyve** [comment]
@nhinsch - RE: the tests issue. I don't think they were passing.

File: tests/integration/serverless.yml (hunk 1 above)
Comment (astuyve) [reply tests/integration/serverless.yml:20]: I inadvertently added it when I wrote the feature, but it includes all headers and payload data that is complex to mask out, so I removed it now.

I think my introduction of this is what led to the tests being in the state they're in now.
